python/items.json

```diff
@@ -0,0 +1 @@
+{"items": [{"name": "jacket", "category": "fashion"}, {"name": "jacket", "category": "fashion"}, {"name": "jacket", "category": "fashion", "image_filename": "image_filename"}, {"name": "jacket", "category": "fashion", "image_filename": "d5bbb87d9f8dd2c43d6bdc98c5e48d6804c770017643a63ec83986ed987bcfb9.jpg"}, {"name": "jacket", "category": "fashion", "image_filename": "d5bbb87d9f8dd2c43d6bdc98c5e48d6804c770017643a63ec83986ed987bcfb9.jpg"}]}
```

Comment:
You can add this file to `.gitignore` also.

Reply:
Added to  ```.gitignore```.